docs/4-guides/1-persistence.mdx

```diff
@@ -56,10 +56,10 @@ We will create a local persistence plugin for IndexedDB on web soon. For React N
 
 ## Roll your own plugin
 
-Local persistence plugins are very simple - all you need is `get`, `set`, and `delete`. See [ObservablePersistLocalStorage](https://github.com/LegendApp/legend-state/blob/main/src/ObservablePersistLocalStorage.ts) for an example of what it looks like. Then you can just pass your provider into `configureObservable`.
+Local persistence plugins are very simple - all you need is `get`, `set`, and `delete`. See [ObservablePersistLocalStorage]([https://github.com/LegendApp/legend-state/blob/main/src/ObservablePersistLocalStorage.ts](https://github.com/LegendApp/legend-state/blob/efda9267211ab771fba1b449f53805691116dad1/src/persist/local-storage.ts#L3)) for an example of what it looks like. Then you can just pass your provider into `configureObservable`.
```

Comment:
Ah good catch, thanks! But it should be `.../blob/main/src/...`?

Reply:
I used the permalink feature, but I did not understand it froze it to the specific version, my bad! Fixed now